docs/photoniq/vwrs/02-get-started-vwrs/index.md

```diff
@@ -0,0 +1,101 @@
+    - **users_per_period:**
+        - Behaves the same as the users access type, except the rate_limit is divided over the defined period.
+        - Rate Calculation Example: If the maximum rate is 100 for 5 seconds, then the rate is 20 users per second.
+    - **rps:**
+        - The metric service counts the number of requests to the origin server per second (RPS).
+        - Rate Calculation Example: If the maximum number of requests allowed is 20 and 13 POST requests are detected, then 7 more requests are released from the waiting room.
+- **rate_period:** Must be set if the access_type is users_per_period.
+    - Defines the time (in seconds) for rate calculation.
+    - Must be a number greater than zero.
+- **rate_limit:** The rate at which traffic should be diverted to the waiting room.
+    - Must be a number greater than zero.
+    
+    Note: **For access_type of users and rps, this is the maximum request rate. For users_per_period, the maximum rate is calculated using the active rate divided by the period**
+
+- **max_origin_usage_time**: The time (in seconds) that users can access the origin after being granted access. After this period, users are forwarded to the waiting room again.
+- **waiting_room_interval**: Defined in seconds, this interval determines how often the waiting room HTML page is updated with information like the request's position, estimated waiting time, and the maximum number of requests in the waiting room.
+
+
+![create virtual waiting room](/img/photoniq/vwr/create-vwr.png)
+
+4. Click **Add Waiting Room**
+
+### Configuring Additional Waiting Room Fields
+
+The Virtual waiting room service allows you to further define and guide the behavior of your waiting room with its **Advanced Settings** fields. These fields include the queue type, dequeue mode, and others.
+
+1. Click **Advanced Settings** from the waiting room definition page
+
+2. Configure these additional details:
+
+![additional details](/img/photoniq/vwr/advanced-settings.png)
+
+- **queue_type**: Defines how requests should be removed from the waiting room. The three possible queue types are fifo, random, and lottery. If this is not set, then the default queue is fifo.
+- **queue_mode**: You can configure the waiting room to be enabled dynamically. When set to auto, the waiting room is enabled after reaching the defined rate_limit for a specific `metric_interval`. If set to manual, then the waiting room is always enabled.
+- **dequeue_mode**: When the value is on, users in the waiting room are granted access to the origin at a rate given by `rate_limit`. When the value is off, users remain in the waiting room indefinitely.
+- **metric_interval**: The time (in seconds) to enable and disable the waiting room. It represents how long the traffic must be at or above the rate limit before being directed to the waiting room.
```

Comment:
@joubert-macrometa should we update this?

Reply:
I updated and avoided listing random and priority as queues. What other updates are needed? I know there's going to be a change in UI, but it has yet to be released. If that's the update, I'll update once the new UI is out.